coordinator/src/main/scala/filodb.coordinator/queryplanner/SingleClusterPlanner.scala

```diff
@@ -134,6 +134,7 @@ class SingleClusterPlanner(val dataset: Dataset,
         many.head match {
           case _: LabelValuesExec => LabelValuesDistConcatExec(qContext, targetActor, many)
           case _: LabelNamesExec => LabelNamesDistConcatExec(qContext, targetActor, many)
+          case _: TsCardExec => TsCardReduceExec(qContext, targetActor, many)
```

Comment:
Where should I cap this off with a presenter? It I do it inside `SingleClusterPlanner`, then `MultiPartitionPlanner` is broken. If I do it in `MultiPartitionPlanner`, then `TsCardinalities` can't be materialized by `SingleClusterPlanners`.

Reply:
deferred to later PR